cogs/displayinfo.py

```diff
@@ -33,85 +32,131 @@ class DisplayInfo(commands.Cog):
 
     def __init__(self,bot):
         self.bot = bot
-        self.timers = {}
-        self.current_collection = ""
+        self.current_collection = db["UserTracking"] # TODO: Move this secrets file
+        self.timers = self.current_collection.find_one() # Gets document from collection.
         self.current_guild_name = ""
         self.current_date = str(date.today())
+        self.members_online = set()
+        self.timed_task = None
+    
+    '''
+    Sample dictionary layout for self.timers: { "user_id" : { 'date1' : { 'Activity1': {'totalTime': [], 'startTime': []}, 'Activity2': {'totalTime': [], 'startTime': []}}}, 
+    'date2': {'Activity1': {'totalTime': [], 'startTime': []}}}
```

Comment:
NIT: Suggestions to tab in the date 2 portion to the same level as date1 and then create user_id2 then ....